### ISSUE TITLE
Add more info to readme

### ISSUE THREAD

**Issue by samfcmc** [Rafael-Baltazar/unblock-solver#1]
What is Unblock game? And why do you want to create a solver for that?

**Comment by Rafael-Baltazar** [comment]
Unblock King: https://play.google.com/store/apps/details?id=com.mobirix.slideking

**Comment by samfcmc** [comment]
I play this game :)
Put the link in the readme file. Do you think that there is an algorithm to solve that game?

**Comment by Rafael-Baltazar** [comment]
Yup. Using backtrack to explore all possible actions. For now, it solves puzzles with only horizontal pieces.

**Comment by samfcmc** [comment]
Can you do that without run out of memory before getting a solution?

**Comment by Rafael-Baltazar** [comment]
It may depend on the level. There are solutions to reduce the memory usage like having a board history that removes actions that were already taken before.
To make sure I get the solution with the least number of steps, I either need to solve all possible actions or know the number of actions in the solution with the least number of steps.
Fortunately, Unblock King tells me the least number of steps (the perfect score) in a solution once I complete the level. With that number I can stop the solving as soon as I get a solution with that number of steps, instead of calculating all possible solutions.

Also I added a new README to the develop branch with information on Unblock King and the solver.

**Comment by samfcmc** [comment]
Ok nice. Btw, Why did you choose JAVA to implement this?

**Comment by Rafael-Baltazar** [comment]
I chose JAVA, because I'm learning Maven.
